### PR TITLE
Clean/file structures

### PR DESCRIPTION
**Pull request by mfunyu**
## What
Updated file structures
- delete: unused files (`go/`, `data/`, `CONTRIBUTING.md`)
- move:  original Build@Mercari `README.md` under `documents/` directory
- add: new `README.md` with install and usage guides
- update: pull request template

### CODE REVIEW

File: document/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Build@Mercari Training Program 2022
+
+This is @mikan373's build training repository.
```
Comment (dominic-soh) [inline document/README.md]: change to team 17?

**Review by dominic-soh — CHANGES_REQUESTED** [review]
Thanks for cleaning up :+1:

File: document/README.md (hunk 1 above)
Comment (mikan373) [reply document/README.md]: That sounds good
Otherwise it LGTM!

Comment (mfunyu) [reply document/README.md]:
```suggestion
This is team17's build training repository.
```

**Pull request merged**